### DISCUSSION TITLE
Can't change the position of a section

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [Lissy93/dashy#1306] [Help and Support]
I have tried different approaches to change the order in which the sections are placed on a screen but cannot get it right. I have tried using the built-in editor as well as edited the confi.yaml file directly but all to no avail.

**Comment by arladmin** [comment]
+1

**Comment by CrazyWolf13** [comment]
https://github.com/Lissy93/dashy/issues/378#issuecomment-999569231
Lissy explained it here.

**Comment by ZuluPilgrim** [comment]
Hi, yes I have tried that. I can move items within a section but not the order of the sections themselves.

**Reply by CrazyWolf13** [reply]
If I remember correctly you can rearrange sections via built-in editor as lissy explained.
Just not via the move section directly in edit mode, but that will be added according to lissy.

**Comment by ZuluPilgrim** [comment]
OK, I think I tried that but must have got something wrong.

All good, I will wait for the added feature :)

Thanks for coming back to me.

André
Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by CrazyWolf13** [reply]
> OK, I think I tried that but must have got something wrong.
> 
> All good, I will wait for the added feature :)
> 
> Thanks for coming back to me.
> 
> André
> Sent with [Proton Mail](https://proton.me/) secure email.
> 
> ------- Original Message -------
> On Wednesday, September 27th, 2023 at 18:43, CrazyWolf13 ***@***.***> wrote:
> 
> > If I remember correctly you can rearrange sections via built-in editor as lissy explained.
> > Just not via the move section directly in edit mode, but that will be added according to lissy.
> >
> > —
> > Reply to this email directly, [view it on GitHub](https://github.com/Lissy93/dashy/discussions/1306#discussioncomment-7119418), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/A2QDWVKDTEL35TTOQU6CYW3X4O4H3ANCNFSM6AAAAAA4FN2T2E).
> > You are receiving this because you authored the thread.Message ID: ***@***.***>

I think for the moment it's best to do it manually as dashy barely got feature updates and was for quite a long time stale and now needs a lot of maintenance, and lissy said she is very busy right now.

Have you ever written to the conf.yml directly?

**Comment by fviccia** [comment]
I just moved it, changing the order of the elements in config json. :)
